File: snippets/csharp/query_point_groups.cs
using Qdrant.Client;

var client = new QdrantClient("localhost", 6334);

await client.QueryGroupsAsync(
	collectionName: "{collection_name}",
    groupBy: "document_id",
    groupSize: 5,
    limit: 10,
	query: new float[] { 0.01f, 0.45f, 0.67f }
);

File: snippets/typescript/query_point_groups.ts
import { QdrantClient } from "@qdrant/js-client-rest";

const client = new QdrantClient({ host: "localhost", port: 6333 });

client.queryGroups("{collection_name}", {
    query: [0.01, 0.45, 0.67],
    group_by: "document_id",
    limit: 10,
    group_size: 5,
});
